internal/biz/model/common_representation.go

```diff
@@ -96,3 +96,13 @@ func DeserializeCommonRepresentation(snapshot *CommonRepresentationSnapshot) Com
 func (cr CommonRepresentation) CreateSnapshot() (CommonRepresentationSnapshot, error) {
 	return cr.Serialize(), nil
 }
+
+// GetField extracts a field from the common representation data
+func (cr CommonRepresentation) GetField(fieldName string) string {
```

Comment:
I would add getters only if needed, if you give access to all the fields this way, we could be breaking encapsulation. You can use this utility method if you want, but make it unexported and only expose the WorkspaceId(). the principle is to hide as much as you can and expose as little as possible.